### PR TITLE
Portfolio: Mobile version work section.

### PR DESCRIPTION
**Pull request by Loltolo-Lesapiti**
In this project, I made the following additions to the portfolio website mobile version.
### Added the about section, which entails the about section and the list of the languages.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        </div>
+         <!---The second card -->
+         <div class="flex">
+            <img src="images/card2-image.png" alt="Image for Card 2">
+            <h2>TONIC</h2>
+           <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="canopy-list"><li>BACK&nbsp;END&nbsp;Dev</li><li>2015</li></ul>
+           </div> 
+            <h4>
```
Comment (danielufeli) [inline index.html]: - [ ] Nice working on your `cards`, kindly use a `p` tag for The paragraph ("A daily selection of privately personalized reads; no accounts or sign-ups required.") for better use of HTML semantics.
- [ ] Also kindly change the `color` and reduce the `font-size` of the list of project technologies to match what you have on the Figma Template.

### Figma Design
is-figma

### Your Design
is-worked

File: index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    <section class="about">
+        <div class="about-content">
+            <h2>About <br> Myself</h2>
+            <p class="my-intro">Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what
+                 you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <br>
+            <p class="lets-connect-message">LET'S CONNECT</p>
```
Comment (danielufeli) [inline index.html]: - [ ] Nice working on you about section, kindly make the `LET'S CONNECT` text a header tag `hx` for better use of HTML Semantics

File: index.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            <ul class="connect-media">
+                <li><a  href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+                <li><a  href="#"><img src="images/Linkedinicon.svg" alt="linkedin"></a></li>
+                <li><a  href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a  href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a  href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+            </ul>
+            <button class="see-more resume">Get my resume</button>
+        </div>
+        <ul class=" about-list">
```
Comment (danielufeli) [inline index.html:110]: - [ ] Nice working on your language section, kindly adjust the `width` and `margin` of the language list so it matches the Figma Template

### Your Design
is-figma-lang

### Figma Design
is-work-lang

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Loltolo-Lesapiti , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️
- Clean Design ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Loltolo-Lesapiti  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**